content/applications/sales/point_of_sale/shop/invoice.rst

```diff
@@ -1,60 +1,63 @@
-==============================
-Invoice from the PoS interface
-==============================
+============================
+Invoices generation from PoS
+============================
 
-Some of your customers might request an invoice when buying from your
-Point of Sale, you can easily manage it directly from the PoS interface.
+In Point of Sale, you can print invoices upon payment at the cash register or retrieve all the past
```

Comment:
"print" alone is not correct here.
It's not nitpicking, as "printing" implies that the invoice already exists, which is not the case.
When you issue an invoice through pos, an invoice is created, the journal items of the sales are modified.

I suggest
... you can issue and print invoices...